R/plotAbundance.R

```diff
@@ -24,36 +24,45 @@
 #'   (Please use \code{assay.type} instead. At some point \code{assay_name}
 #'   will be disabled.)
 #'   
-#' @param features a single \code{character} value defining a column from 
+#' @param cor.var a single \code{character} value defining a column from 
```

Comment:
features is not good name since usually we mean taxa/rows with features.

However, this cor.var sounds like "variable that is analyzed for correlation" or something. col.var might be better --> "column variable".